src/main/scripts/org/reaktivity/specification/amqp/session/incoming.window.exceeded/server.rpt

```diff
@@ -152,10 +147,11 @@ read [0x00 0x00 0x00 0x23]                                        # length = 35
      [0xa0 0x01 0x31]                                             # delivery-tag = "1"
      [0x43]                                                       # message-format = 0
      [0x41]                                                       # settled = true
-     [0x00 0x53 0x77 0xb0 0x00 0x00 0x00 0x05] "Text!"            # payload = "Text!"
+     [0x00 0x53 0x77 0xb0 0x00 0x00 0x00 0x05]                    # message format = amqp-value, binary4, size = 5
+     "Text!"                                                      # body = "Text!"
```

Comment:
If the message is `"Text!"`, should the message format not be `str8-utf8` text instead of binary?